src/Tickets/Seating/Commerce/Controller.php

```diff
@@ -34,6 +39,9 @@ protected function do_register(): void {
 			'tec_tickets_seating_timer_token_object_id_entries',
 			[ $this, 'filter_timer_token_object_id_entries' ],
 		);
+
+		add_filter( 'tec_tickets_commerce_increase_ticket_stock', [ $this, 'sync_seated_tickets_stock' ], 10, 3 );
```

Comment:
As suggested in my other comment, I would generalize this by watching for updates to the `_stock` meta key.